src/transformers/tokenization_t5.py

```diff
@@ -145,6 +146,30 @@ def get_vocab(self):
         vocab.update(self.added_tokens_encoder)
         return vocab
 
+    def build_inputs_with_special_tokens(self, token_ids_0: List[int], token_ids_1: Optional[List[int]] = None) -> List[int]:
+        """
+        Build model inputs from a sequence or a pair of sequence for sequence classification tasks
+        by concatenating and adding special tokens.
+        A sequence has the following format:
+
+        - single sequence: ``<s> X </s>``
+        - pair of sequences: ``<s> A </s></s> B </s>``
```

Comment:
@sshleifer - Could it be that this is a bug? `[BOS] A [EOS][EOS] B [EOS]` seems a bit weird to me, but not sure -> 
I would have guessed it should be `[BOS] A [EOS][BOS] B [EOS]` .